fableous-fe/src/containers/HubCanvasPage.tsx

```diff
@@ -608,28 +694,70 @@ export default function HubCanvasPage() {
           hubState !== HubState.DrawingSession && "invisible"
         }`}
       >
-        <Grid container>
+        <Grid container className="mt-4">
           <Grid item xs={12}>
             <ChipRow
               primary
               chips={[
                 <Chip label={story?.title} color="primary" />,
-                ...(story?.description.split(",") || []),
+                <div className="flex gap-4">
+                  {(story?.description.split(",") || []).map((tag) => (
+                    <Chip label={tag} color="secondary" />
+                  ))}
+                </div>,
               ]}
             />
           </Grid>
         </Grid>
-        <Grid container className="flex-1 my-4">
-          <Grid item xs={12}>
+        <Grid container spacing={2} className="flex-1 my-4">
```

Comment:
it shows a scrollbar on my localhost, apply styling similar to hideScrollbar in CanvasToolbar

also do we want to limit the width as well? CanvasToolbar currently has max width 80px

![image](https://user-images.githubusercontent.com/29034900/134647704-909470b9-ae45-4bf5-aedc-b2c4573d3823.png)

Reply:
maxWidth for layer toolbar was initially set to 100. will revert